tendrl/gluster_integration/sds_sync/georep_details.py

```diff
@@ -188,19 +188,38 @@ def aggregate_session_status():
                     continue
                 pair_count = 0
                 faulty_count = 0
+                stopped_count = 0
+                paused_count = 0
+                created_count = 0
                 for element in pairs.leaves:
                     pair = GeoReplicationPair(
                         vol_id=vol_id,
                         session_id=session_id,
                         pair=element.key.split("pairs/")[-1]
                     ).load()
-                    if pair.status == "Faulty":
+                    if pair.status.lower() == "faulty":
                         faulty_count += 1
+                    elif pair.status.lower() == "created":
+                        created_count += 1
+                    elif pair.status.lower() == "stopped":
+                        stopped_count += 1
+                    elif pair.status.lower() == "paused":
+                        paused_count += 1
                     pair_count += 1
```

Comment:
@shtripat as per aravinda, If a particular node is down then the pair info for bricks in the node will not appear as part of get state. which might lead to below logic going wrong. So its better to find the pair count by counting the number of bricks in master volume (because nos of  pairs in geo-rep session = nos of bricks in master volume).

Reply:
Yes. thats a good point. Will change the logic accordingly.